src/brs/peer/PeerImpl.java

```diff
@@ -142,7 +142,7 @@ void setVersion(String version) {
     if (Burst.APPLICATION.equals(getApplication()) && version != null) {
       try {
         this.version.set(Version.parse(version));
-        isOldVersion.set(Constants.MIN_VERSION.isGreaterThan(this.version.get()));
+        isOldVersion.set(Burst.getMinVersion().isGreaterThan(this.version.get()));
```

Comment:
Min version is not a constant anymore but a variable depending on block height, so a method call